packages/core/admin/admin/src/content-manager/components/Inputs/index.js

```diff
@@ -224,6 +224,8 @@ function Inputs({
       attribute={fieldSchema}
       autoComplete="new-password"
       intlLabel={{ id: label, defaultMessage: label }}
+      // in case the default value of the boolean is null, attribute.default doesn't exist
+      isNullable={inputType === 'bool' && fieldSchema.attribute?.default === undefined}
```

Comment:
`inputType === 'bool' && fieldSchema.default=== undefined`

Reply:
Thanks. It was more of a remark to me, so I don't forget it. Fixed.